components/textbook-beta/SketchedLogo.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <svg
+      class="logo-fill"
```

Comment:
Small change follow our BEM convention:

```suggestion
      class="logo__fill"
```